src/swap.flows/ETH2BTC.js

```diff
@@ -248,9 +249,11 @@ class ETH2BTC extends Flow {
           return
         }
 
+        console.log("Debug - destination address....")
```

Comment:
```suggestion
        debug("Debug - destination address....")
```
сделай rebase на release-0.3.1, я думаю.

и тогда можно будет использовать debug для этого